spec/index.html

```diff
@@ -5669,248 +5670,259 @@ <h5>NOT IN</h5>
         </section>
         <section id="func-rdfTerms">
           <h4>Functions on RDF Terms</h4>
-          <section id="func-RDFterm-equal">
-            <h5>RDFterm-equal</h5>
-            <pre class="prototype nohighlight">
-              <span class="return">xsd:boolean</span> <span class="operator" style="text-transform: none;">RDFterm-equal</span> (<span class="type"><span class="type RDFterm">RDF term</span></span> <span class="name">term1</span>, <span class="type"><span class="type RDFterm">RDF term</span></span> <span class="name">term2</span>)
-            </pre>
-            <p>This function cannot be used directly in expressions. The purpose of this function is to define the semantics of the "=" operator when applied to two RDF terms that do not fall into any of the other, more concrete cases covered in the operator mapping table in Section&nbsp;<a href="#OperatorMapping" class="sectionRef"></a>.</p>
-            <p>The function is defined as follows:</p>
-            <ul>
-                <li>Returns TRUE if <code>term1</code> and <code>term2</code> are <em>equal</em> RDF terms, as defined below.</li>
-                <li>Produces a type error if <code>term1</code> and <code>term2</code> are both literals having the
-              same <a data-cite="RDF12-CONCEPTS#dfn-datatype-iri">datatype IRI</a>; this datatype IRI is <em>not</em> in the
-              set of <a data-cite="RDF12-CONCEPTS#dfn-recognized-datatype-iri">recognized datatype IRIs</a>; and the
-              <a data-cite="RDF12-CONCEPTS#dfn-lexical-form">lexical forms</a> of the two literals are different from one another.</li>
-                <li>Returns FALSE otherwise.</li>
-            </ul>
+
+          <section id="func-sameTerm">
+            <h5>sameTerm</h5>
+            <pre class="prototype nohighlight"> <span class="return">xsd:boolean</span>  <span class="operator">sameTerm</span> (<span class="type"><span class="type RDFterm">RDF term</span></span> <span class="name">term1</span>, <span class="type"><span class="type RDFterm">RDF term</span></span> <span class="name">term2</span>)</pre>
+            <p>Returns TRUE if <code>term1</code> and <code>term2</code> are the same RDF term as
+              defined in [[[RDF12-CONCEPTS]]] [[RDF12-CONCEPTS]]; returns FALSE otherwise.</p>
             <p>
-              <code>term1</code> and <code>term2</code> are <em>equal</em> RDF terms if any of the following is
-              true:</p>
+              |term1| and |term2| are the
+              <a data-cite="RDF12-CONCEPTS#dfn-rdf-term-equality">same RDF terms</a>
```

Comment:
```suggestion
              In particular, |term1| and |term2| are the
              <a data-cite="RDF12-CONCEPTS#dfn-rdf-term-equality">same RDF term</a>
```

Reply:
Change to "same RDF term".

The "In particular" reads to me as a focus on some specific case, whereas this is reiterating all of the RDF Concepts definition.